kukkuu/urls.py

```diff
@@ -23,8 +23,8 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path(
-        "graphql",
+    re_path(
+        "graphql/?$",
```

Comment:
I think `r"^graphql/?$"` would be more the usual way.

Reply:
"Fixed" :)